index.js

```diff
@@ -24,64 +26,93 @@ function getRandomNumber(min, max) {
 
 function checkGuess() {
   // Get value from guess input element
-  const guess = parseInt(guessInput.value, 10);
-  attempts = attempts + 1;
-
+  let guess = parseInt(guessInput.value, 10);
+  guessInput.value='';
+  
   hideAllMessages();
-
+  resetButton.style.display = ''
+  
   if (guess === targetNumber) {
-    numberOfGuessesMessage.style.display = '';
     numberOfGuessesMessage.innerHTML = `You made ${attempts} guesses`;
-
+    numberOfGuessesMessage.style.display='block';
     correctMessage.style.display = '';
-
     submitButton.disabled = true;
+    
     guessInput.disabled = true;
+    
+    resetButton.style.display = '';
+    
   }
+  
+   
 
-  if (guess !== targetNumber) {
-    if (guess < targetNumber) {
-      tooLowMessage.style.display = '';
-    } else {
-      tooLowMessage.style.display = '';
-    }
+  if (guess < targetNumber) {
+    tooLowMessage.style.display = '';
+    
+  }
+  
+   
+   
+  if (guess > targetNumber) {
+    tooHighMessage.style.display = '';
+    
+  }
 
-    const remainingAttempts = maxNumberOfAttempts - attempts;
 
-    numberOfGuessesMessage.style.display = '';
-    numberOfGuessesMessage.innerHTML = `You guessed ${guess}. <br> ${remainingAttempts} guesses remaining`;
-  }
+  const remainingAttempts = maxNumberOfAttempts - attempts;
+  console.log(remainingAttempts);
 
-  if (attempts ==== maxNumberOfAttempts) {
-    submitButton.disabled = true;
-    guessInput.disabled = true;
+  if(guess !==targetNumber){
+    
+
+    maxGuessesMessage.innerHTML = `You guessed ${guess} <br>   ${remainingAttempts} guesses remaining`;
+  maxGuessesMessage.style.display = '';
+  
+    
+}
+
+else{
+  document.getElementById('max-guesses');
+  
+  //submitButton.disabled=true;
+   //guessInput.disabled=true;
+ }
```

Comment:
Is this `else` statement still needed; since there's nothing being done inside it?